services/traction/conftest.py

```diff
@@ -31,7 +32,7 @@ async def db_session() -> AsyncSession:
         async with async_session(bind=connection) as session:
             yield session
             await session.flush()
-            await session.rollback()
+            await session.commit()
```

Comment:
rolling back is what is supposed to keep each test isolated from the next. 

Shouldn't using the `app_client` fixture below give you the desired effect of committing the data changed by the test?